### PR TITLE
Adding the ability to change/add labels in service monitor

### PR DESCRIPTION
**Pull request by liranmauda**
Adding the ability to change/add labels in service monitor

Signed-off-by: liranmauda <liran.mauda@gmail.com>

### CODE REVIEW

**Comment by liranmauda** [comment]
@umangachapagain @anmolsachan Can you review?

**Comment by anmolsachan** [comment]
Looks good in principle.

I would request you to create the corresponding PR on the OCS operator to pass the monitoring label and test it all together with ODF on an OCP cluster.

File: pkg/system/phase4_configuring.go [hunk 1]
```diff
@@ -22,6 +22,9 @@ import (
 	"github.com/noobaa/noobaa-operator/v5/pkg/nb"
 	"github.com/noobaa/noobaa-operator/v5/pkg/options"
 	"github.com/noobaa/noobaa-operator/v5/pkg/util"
+
+	// monitoringv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
```
Comment (umangachapagain) [inline pkg/system/phase4_configuring.go:26]: Do we need to preserve this?

Comment (liranmauda) [reply pkg/system/phase4_configuring.go:26]: Correct, It was missed by me, Thanks 🙂

**Review by dannyzaken — APPROVED** [review]

**Pull request merged**